Automattic/wp-calypso
# Reader Stream Refresh: add action links block to stream card

Add the new `ReaderPostActions` block to the new stream card.

<img width="777" alt="screen shot 2016-10-13 at 14 11 06" src="https://cloud.githubusercontent.com/assets/17325/19333099/0478b28a-914f-11e6-9468-54d48eae3b41.png">

## To test

Load your following stream at http://calypso.localhost:3000/ and make sure that the action buttons are correctly aligned and function as expected.
